experiment/generate_tests.py

```diff
@@ -274,13 +233,6 @@ def generate(self):
         k8s_version = self.k8s_versions[fields[5][3:]]
         test_suite = self.test_suites[fields[6]]
 
-        # Generates envs.
-        envs = []
-        envs.extend(get_envs(self.job_name, 'common', self.common))
```

Comment:
We can remove this part because no configs use envs?

Reply:
I believe those envs are written out as the `--env-file=job.env`? I've checked nothing writes out env files anymore.

Reply:
You are right. I was thinking about the `--env` flags, but I think they are handled as `args`.